service/scheduler/api/src/main/java/org/eclipse/kapua/service/scheduler/trigger/TriggerService.java

```diff
@@ -40,4 +40,6 @@
      */
     public TriggerListResult query(KapuaQuery<Trigger> query)
             throws KapuaException;
+
+    public void setQuery(KapuaQuery<Trigger> query) throws KapuaException;
```

Comment:
This method MUST be removed from the API. 
Service APIs MUST remain clean, avoiding adding 'in-utility' methods.